toqito/state_opt/state_distinguishability.py

```diff
@@ -69,61 +72,83 @@ def state_distinguishability(
     >>> e_11 = e_1 * e_1.conj().T
     >>> states = [e_00, e_11]
     >>> probs = [1 / 2, 1 / 2]
-    >>> res = state_distinguishability(states, probs)
+    >>> res, _ = state_distinguishability(states, probs)
```

Comment:
I think it might be better to add a few examples that show off the different solver options: `primal_dual`. A user has the choice of `dual` or `primal`. 

This might be a better option as the private functions  `_min_error_primal` and ` _min_error_dual` are not going to show up in the docs build.

Also, for the existing example, make it obvious somehow that we are using the default option of `dual`.

Reply:
Yep, added some additional details in the docstring.